fairscale/nn/data_parallel/fully_sharded_data_parallel.py

```diff
@@ -321,6 +321,7 @@ def __init__(
         offload_config: Optional[OffloadConfig] = None,
         state_dict_on_rank_0_only: bool = False,
     ):
+        torch._C._log_api_usage_once("fairscale.fsdp")
```

Comment:
Check torch version before calling this?

Reply:
Thank you @min-xu-ai , I will do try-except and only logging if _C module is available